public/javascripts/Admin/src/Admin.User.js

```diff
@@ -47,6 +47,13 @@ function AdminUser(params) {
             }
         })
             .addTo(map);
+
+            // Calculate total distance audited
+            var distanceAudited = 0;
+            for (var i = data.features.length - 1; i >= 0; i--) {
+                distanceAudited += turf.length(data.features[i], {units: 'miles'});
```

Comment:
This code was actually just rewritten slightly in Progress.js so that it shows in as kilometers instead of miles if we aren't using US English in the browser. Could you update yours to work the same way?